loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/FlowOutputWriter.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.loadflow.validation;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+public enum FlowOutputWriter {
+    CSV(FlowsFormatterCsvWriterFactory.class),
+    CSV_PROPERTIES(FlowsFormatterCsvPropertiesWriterFactory.class);
```

Comment:
I'm not sure the enum values are well named. CSV is ok for me. The difference between the two output formats is mono-line vs multi-line